src/app/file-manager/transferables-grid/transferables-grid.component.html

```diff
@@ -81,9 +94,9 @@ <h4 class="subtitle">Exported: {{filesDatabase.exportingCount + ' of ' + filesDa
         <mat-cell *matCellDef="let row">
           <mat-icon class="transfer-icon" *ngIf="row.type == 'Download'">file_download</mat-icon>
           <mat-icon class="transfer-icon" *ngIf="row.type == 'Upload'">file_upload</mat-icon>
-          <i class="export-icon-dark" *ngIf="row.type == 'Export GCP'"></i>
+          <i class="export-icon-dark" *ngIf="row.type == 'Export GCP' || row.type == 'Export S3' "></i>
           {{row.status}}
-          <div *ngIf="row.status === 'Downloading' || row.status === 'Uploading' || row.status === 'Exporting to GCP'" class="progress-dots transfer-status">
+          <div *ngIf="row.status === 'Downloading' || row.status === 'Uploading' || row.status === 'Exporting to Google Cloud' || row.status === 'Exporting to Amazon'" class="progress-dots transfer-status">
```

Comment:
Relying on text like this seems somewhat fragile, as the wording might change at some point. Is there a better way to do this, e.g. using a fixed token in the row that's not visible in the UI to control this?

Reply:
I think the best for this is create a separate ticket.